src/plugins/dashboard/public/dashboard_api/panels_manager.ts

```diff
@@ -419,31 +412,18 @@ export function initializePanelsManager(
         references: Reference[];
       }> => {
         const references: Reference[] = [];
```

Comment:
you can remove `async` from `getState` method

Reply:
removed in 8b0809cccd7